test/e2e/opaque_resource.go

```diff
@@ -38,7 +38,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = framework.KubeDescribe("Opaque resources [Feature:OpaqueResources]", func() {
+var _ = framework.KubeDescribe("Opaque resources [Serial]", func() {
```

Comment:
why should it be serial? If these tests are only adding/removing/modifying OIRs to nodes, why would it impact other tests that dont deal with OIRs? IOW why adding additional resource types to nodes would impact other tests that dont deal with the new types?